### PR TITLE
Use tenant and name parameters when render

### PR DESCRIPTION
**Pull request by lukasz-kaniowski**
When we render templates corectl passes app `name` and `tenant` as parameters. More details https://github.com/coreeng/corectl/pull/26

This PR is using those values to parametrized rendered templates

### CODE REVIEW

File: .github/workflows/trigger-render.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ jobs:
     steps:
       - uses: actions/github-script@v6
         with:
-          github-token: ${{ secrets.TEMPLATES_RENDER_GH_TOKEN }}
+          github-token: {% raw %}${{ secrets.TEMPLATES_RENDER_GH_TOKEN }}{% endraw %}
```
Comment (Mugenor) [inline .github/workflows/trigger-render.yaml:14]: This is unnecessary, isn't it?

File: java/web/skeleton/Makefile [hunk 2]
```diff
@@ -1,7 +1,7 @@
 projectDir := $(realpath $(dir $(firstword $(MAKEFILE_LIST))))
 os := $(shell uname)
-app_name = java-web
-tenant_name = $(TENANT_NAME)
+app_name = {{ name | default("java-web") }}
+tenant_name = {{ tenant | default("$(TENANT_NAME)") }}
```
Comment (Mugenor) [inline java/web/skeleton/Makefile:4]: Should helm chart also be updated similar to go template?
Also, does it make sense to update project name in `settings.gradle` and similar files?

File: .github/workflows/trigger-render.yaml (hunk 1 above)
Comment (lukasz-kaniowski) [reply .github/workflows/trigger-render.yaml:14]: yep, thanks

File: java/web/skeleton/Makefile (hunk 2 above)
Comment (lukasz-kaniowski) [reply java/web/skeleton/Makefile:4]: I think I've updated helm charts, but will double check. Will have a look in settings.gradle

Comment (lukasz-kaniowski) [reply java/web/skeleton/Makefile:4]: ah, I've done go-web but not java-web

File: go/web/skeleton/Makefile [hunk 3]
```diff
@@ -1,7 +1,7 @@
 projectDir := $(realpath $(dir $(firstword $(MAKEFILE_LIST))))
 os := $(shell uname)
-app_name = go-web
-tenant_name = $(TENANT_NAME)
+app_name = {{ name | default("go-web") }}
+tenant_name = {{ tenant | default("$(TENANT_NAME)") }}
```
Comment (tombart) [inline go/web/skeleton/Makefile:4]: do we have docs saying that we can set that variable and the tenant will be populated?

**Review by tombart — APPROVED** [review]

File: go/web/skeleton/Makefile (hunk 3 above)
Comment (lukasz-kaniowski) [reply go/web/skeleton/Makefile:4]: `corectl app create -h` mentions it, but will add some more documentation to docs once I'm done

**Pull request merged**